.github/workflows/ci.yml

```diff
@@ -248,6 +248,9 @@ jobs:
               env:
                   DATABASE_URL: mysql://root:root@127.0.0.1:3306/contao_test
 
+            - name: Run the unit tests in reverse order
+              run: vendor/bin/phpunit --order-by=reverse --colors=always
+
```

Comment:
We should probably create an extra job that only runs in the newest php version.